letsencrypt-nginx/letsencrypt_nginx/parser.py

```diff
@@ -215,7 +215,7 @@ def filedump(self, ext='tmp'):
             try:
                 with open(filename, 'w') as _file:
                     nginxparser.dump(tree, _file)
-            except IOError:
+            except IOError: # pragma: no cover
```

Comment:
It seems a shame to no-cover these. We haven't done that for exception handlers elsewhere in the codebase. Perhaps we can find a way to exercise them without too much painful work.